conf/app.routes

```diff
@@ -1,6 +1,7 @@
 # microservice specific routes
 
-GET       /                                               uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.index()
+GET       /                                               uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.legacy()
+GET       /new                                            uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.index()
```

Comment:
Yeah.. had wondered the same thing tbh. What I had been thinking was this would go into prod temporarily as the gateway to the old calc pages, and I wasn't sure whether the convention for the default landing pages in an app for the LB was the app default index page or specified in the nginx config ? Ideas ?